whittle/search/local_search.py

```diff
@@ -27,21 +27,21 @@ class PopulationElement:
 
 class LS(FIFOScheduler):
     """
-
     See :class:`~syne_tune.optimizer.schedulers.searchers.RandomSearcher`
-    for ``kwargs["search_options"]`` parameters.
-
-    :param config_space: Configuration space for evaluation function
-    :param metric: Name of metric to optimize
-    :param population_size: See
-        :class:`~syne_tune.optimizer.schedulers.searchers.RegularizedEvolution`.
-        Defaults to 100
-    :param sample_size: See
-        :class:`~syne_tune.optimizer.schedulers.searchers.RegularizedEvolution`.
-        Defaults to 10
-    :param random_seed: Random seed, optional
-    :param kwargs: Additional arguments to
-        :class:`~syne_tune.optimizer.schedulers.FIFOScheduler`
+       for ``kwargs["search_options"]`` parameters.
+
+       Args:
+           config_space: Configuration space for evaluation function
+           metric: Name of metric to optimize
+           population_size: See
+               :class:`~syne_tune.optimizer.schedulers.searchers.RegularizedEvolution`.
```

Comment:
Do these links to different classes actually work with our mkdocs configuration?

Reply:
they are not shown as links

Reply:
I didn't write this docstring initially, I just reformulated how it was written.

Reply:
ping @aaronkl

Reply:
we can remove these links